Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -2395,6 +2395,41 @@ class PaymentSheetLinkUITests: PaymentSheetUITestCase {
 //    }
 }
 
+class PaymentSheetDefaultSPMUITests: PaymentSheetUITestCase {
+    func testDefaultSPMHorizontalNavigation() {
```

Comment:
This is good! Maybe add the inverse for when defaultSPMNavigation is off and we only get 1 circular edit button.